scripts/hid_configurator/configurator_cli.py

```diff
@@ -177,6 +199,8 @@ def configurator():
         perform_fwreboot(dev, args)
     elif args.command == 'config':
         perform_config(dev, args)
+    elif args.command == 'led_stream':
```

Comment:
What about change of if-else sequence to single call ``allowed_command[args.commnad](dev,args)``

Reply:
If we stick to passing `args` like that, then this could be good solution.